test/unit/gateways/simetrik_test.rb

```diff
@@ -23,7 +22,84 @@ def setup
     @trace_id = SecureRandom.uuid
     @order_id = SecureRandom.uuid[0..7]
 
-    setup_options()
+    @sub_merchant = {
+      address: 'string',
+      extra_params: {},
+      mcc: 'string',
+      merchant_id: 'string',
+      name: 'string',
+      phone_number: 'string',
+      postal_code: 'string',
+      url: 'string'
+    }
+
+    @authorize_capture_options = {
+      acquire_extra_options: {},
+      trace_id: @trace_id,
+      user: {
+        id: '123',
+        email: 's@example.com'
+      },
+      order: {
+        id: @order_id,
+        datetime_local_transaction: @datetime,
+        description: 'a popsicle',
+        installments: 1,
+        amount: {
+          currency: 'USD',
+          vat: 19
+        }
+      },
+      three_ds_fields: {
+        version: '2.1.0',
+        eci: '02',
+        cavv: 'jJ81HADVRtXfCBATEp01CJUAAAA',
+        ds_transaction_id: '97267598-FAE6-48F2-8083-C23433990FBC',
+        acs_transaction_id: '13c701a3-5a88-4c45-89e9-ef65e50a8bf9',
+        xid: '00000000000000000501',
+        enrolled: 'string',
+        cavv_algorithm: '1',
+        directory_response_status: 'Y',
+        authentication_response_status: 'Y',
+        three_ds_server_trans_id: '24f701e3-9a85-4d45-89e9-af67e70d8fg8'
+      },
+      sub_merchant: @sub_merchant,
+      token_acquirer: @token_acquirer
+    }
+
+    @authorize_capture__fail_options = {
```

Comment:
Is there any difference between this variable and the `@authorize_capture_options`? Do we need both?

Reply:
Thanks for letting us know, there was no difference and the redundant variable has been removed.